tests/Sniffs/Variables/data/duplicateAssignmentToVariableErrors.php

```diff
@@ -4,3 +4,6 @@
 
 $b = true;
 $b = $b;
+
+/** @var int $c */
+$c = $c;
```

Comment:
@WinterSilence This line SHOULD NOT be reported by the sniff.

Reply:
@kukulich for reason?

Reply:
Because these lines exist to fix issues reported by static analysis so these lines are valid and should not be reported.

Reply:
@kukulich ok, i move it other file and run `phpcs`:
~~~
FILE: ./sandbox/github/coding-standard/errors.php
---------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 6 ERRORS AFFECTING 5 LINES
---------------------------------------------------------------------------------------------------------------------------------------------------------
  1 | ERROR | [x] Missing declare(strict_types = 1). (SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing)
  3 | ERROR | [ ] Duplicate assignment to variable $a. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  3 | ERROR | [ ] Duplicate assignment to variable $a. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  6 | ERROR | [ ] Duplicate assignment to variable $b. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  9 | ERROR | [x] Use assertion instead of inline documentation comment.
    |       |     (SlevomatCodingStandard.PHP.RequireExplicitAssertion.RequiredExplicitAssertion)
 10 | ERROR | [ ] Duplicate assignment to variable $c. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
---------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
~~~
can you post full example?

Reply:
@kukulich 
~~~
FILE: ./sandbox/test.php
---------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 6 ERRORS AFFECTING 5 LINES
---------------------------------------------------------------------------------------------------------------------------------------------------------
  1 | ERROR | [x] Missing declare(strict_types = 1). (SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing)
  3 | ERROR | [ ] Duplicate assignment to variable $a. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  3 | ERROR | [ ] Duplicate assignment to variable $a. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  6 | ERROR | [ ] Duplicate assignment to variable $b. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
  9 | ERROR | [x] Use assertion instead of inline documentation comment.
    |       |     (SlevomatCodingStandard.PHP.RequireExplicitAssertion.RequiredExplicitAssertion)
 10 | ERROR | [ ] Duplicate assignment to variable $c. (SlevomatCodingStandard.Variables.DuplicateAssignmentToVariable.DuplicateAssignment)
~~~